awx/main/management/commands/list_instances.py

```diff
@@ -13,7 +13,7 @@ class Ungrouped(object):
 
     @property
     def instances(self):
-        return Instance.objects.filter(rampart_groups__isnull=True)
+        return Instance.objects.filter(rampart_groups__isnull=True).exclude(node_type='hop')
```

Comment:
I don't think an IG can have a hop node assigned to begin with. To me, it seems reasonable that the command would list them, although maybe in their own category. It might be good to file an issue instead of tacking it right now.